### ISSUE TITLE
Pulmonary nodule

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6633]
**Preferred term label:** Pulmonary nodule

**Synonyms**

**Definition (free text, please give PubMed ID)**
Focal rounded or ovoid opacity, ≤ 3 cm in diamter

**Parent term (use hpo.jax.org/app)**
Pulmonary interstitial high-resolution computed tomography abnormality HP:0025389

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by eswietlik** [comment]
@pnrobinson Would be great to discuss lung nodules topic with Matthias tomorrow.

**Comment by pnrobinson** [comment]
added
